### PR TITLE
Allow to optionally export total averages of sla reports

### PR DESCRIPTION
**Pull request by jhoxhaa**

### CODE REVIEW

File: library/Icingadb/ProvidedHook/Reporting/SlaReport.php [hunk 1]
```diff
@@ -188,6 +188,12 @@ public function initConfigForm(Form $form)
             'min'         => '1',
             'max'         => '12'
         ]);
+
+        $form->addElement('checkbox', 'csv-json-export', [
```
Comment (lippserd) [inline library/Icingadb/ProvidedHook/Reporting/SlaReport.php:192]: For field names we always use `snake_case`. I would also suggest to rename the field to `export_total`.

File: library/Icingadb/ProvidedHook/Reporting/SlaReport.php [hunk 2]
```diff
@@ -188,6 +188,12 @@ public function initConfigForm(Form $form)
...
+            'label'          => 'Export total for CSV and JSON',
```
Comment (lippserd) [inline library/Icingadb/ProvidedHook/Reporting/SlaReport.php:193]: `Export total values to CSV and JSON`

File: library/Icingadb/ProvidedHook/Reporting/SlaReport.php [hunk 3]
```diff
@@ -188,6 +188,12 @@ public function initConfigForm(Form $form)
...
+            'checkedValue'   => '1',
```
Comment (lippserd) [inline library/Icingadb/ProvidedHook/Reporting/SlaReport.php:194]: Can you add a comment with a little explanation why we're setting `0` and `1` here?

**Review by lippserd — CHANGES_REQUESTED** [review]

File: library/Icingadb/ProvidedHook/Reporting/SlaReport.php [hunk 4]
```diff
@@ -188,6 +188,13 @@ public function initConfigForm(Form $form)
             'min'         => '1',
             'max'         => '12'
         ]);
+
+        $form->addElement('checkbox', 'export_total', [
+            'label'          => 'Export total for CSV and JSON',
```
Comment (lippserd) [inline library/Icingadb/ProvidedHook/Reporting/SlaReport.php:193]: Translate missing :).

**Review by lippserd — CHANGES_REQUESTED** [review]

**Comment by yhabteab** [comment]
@lippserd please review & merge this PR.

**Review by lippserd — APPROVED** [review]

**Pull request merged**